docs/adrs/9999-use-playwright.md

```diff
@@ -0,0 +1,82 @@
+
+Like other third-party code libraries, Cypress recieves periodic version updates from its maintainers, in order to improve performance and/or implement bug fixes; these updates may include security fixes. For the past year, our project has been stuck with using version 8.5.0 (released in [September 2021](https://docs.cypress.io/guides/references/changelog#8-5-0)), as using any further versions in our project caused an elevated rate of test flakiness, to the point where it would functionally hinder or block development.
+
+In July 2022, an effort was undertaken to [upgrade the version of Cypress to 10.3.1](https://github.com/transcom/mymove/pull/8934), which was the current version at the time. It was discovered that, with the upgrade to version 10, a non-trivial amount of file renaming and restructuring work would need to be performed; once that was completed, [the rate of test failures increased significantly](https://docs.google.com/spreadsheets/d/1AsBB8_EkQ1ltqM2a4zfdDs4EvTfqdN76slUrdXeH5LA/edit#gid=0). Several existing and currently passing tests were now failing on each execution, with others now failing at an incidence rate of 10-20% when run as a part of our CI process. This was investigated, but ultimately set aside in the interest of feature development work.
+
+In December 2022, a renewed effort was undertaken to [upgrade the version of Cypress to 12.0.2](https://github.com/transcom/mymove/pull/9737), which was now the current version. The same file renaming and restructuring work needed to be performed; once finished, [the rate of test failures skyrocketed](https://docs.google.com/spreadsheets/d/1P2qvEZqdW9Wyg5McYAe7GNt64abSKjSU3pFhIsJneZE/edit#gid=1060625568). The number of both consistently failing and sporadically failing tests had approximately doubled from the effort six months earlier, and now affected at least half of all integration tests in our existing suite.
+
+The widely accepted explanation for all of the sporadic test failures (and a significant portion of the consistent failures) is that our integration test code has been written in a way that creates race conditions that are experienced when executed; it's likely that the upgraded version of the Cypress library contains a faster runtime that exposes those race conditions to a higher degree. As the code within our test suite has evolved over a long period and many contributors to it are no longer on the project, the chances of understanding _why_ it was written in the way it was seems impossible.
+
+## Considered Alternatives
+
+- Continue using existing version of Cypress
+- Upgrade to current version of Cypress
+- Convert integration tests to Playwright, gradually
+- **Convert integration tests to Playwright, all at once**
+
+## Decision Outcome
+
```

Comment:
**suggestion:** Can we add a line that states the decision outcome? e.g. "Convert Integration Tests to Playwright All At Once"